lib/src/airship_push.dart

```diff
@@ -16,15 +16,30 @@ class AirshipPush {
         this._module = module, this.iOS = IOSPush(module);
 
   /// Tells if user notifications are enabled or not.
-  Future<bool?> get userNotificationsEnabled async {
-    return await _module.channel.invokeMethod('push#getUserNotificationsEnabled');
+  Future<bool?> get isUserNotificationsEnabled async {
+    return await _module.channel.invokeMethod('push#isUserNotifictionsEnabled');
   }
 
   /// Enables or disables the user notifications.
   Future<bool?> setUserNotificationsEnabled(bool enabled) async {
     return await _module.channel.invokeMethod('push#setUserNotificationsEnabled', enabled);
   }
 
+  /// Enables user notifications.
+  Future<bool?> enableUserNotifications() async {
+    return await _module.channel.invokeMethod('push#enableUserNotifications');
+  }
+
+  /// Gets the notification status.
+  Future<bool?> getNotificationStatus() async {
+    return await _module.channel.invokeMethod('push#getNotificationStatus');
+  }
+
```

Comment:
Why do we have different patterns ie: `getRegistrationToken()` `getNotificationStatus()  ` but also `get badge` ?

Reply:
I think it's because the badge can be set, that's why I put a simple getter
Meanwhile we can't manually set the registrationToken or the NotificationStatus

But if you think it's better to standardize all the getters in some way, I could to that